src/components/forms/AttributeForm.jsx

```diff
@@ -0,0 +1,182 @@
+import { makeStyles } from '@material-ui/core/styles';
+import { useTranslation } from 'react-i18next';
+import StringField from '../input/StringField';
+import BoolField from '../input/BoolField';
+import DateTimeField from '../input/DateTimeField';
+import IntegerField from '../input/IntField';
+import FloatField from '../input/FloatField';
+import { IconButton, InputAdornment, Tooltip } from '@material-ui/core';
+import { Clear as ClearIcon } from '@material-ui/icons';
+import Card from '@material-ui/core/Card';
+import CardContent from '@material-ui/core/CardContent';
+import CardHeader from '@material-ui/core/CardHeader';
+import Key from '@material-ui/icons/VpnKey';
+import Attributes from '@material-ui/icons/HdrWeakTwoTone';
+import Typography from '@material-ui/core/Typography';
+import Grid from '@material-ui/core/Grid';
+const useStyles = makeStyles((theme) => ({
+  root: {
+    margin: '0rem',
+  },
+  card: {
+    margin: '0rem',
+    maxHeight: '70vh',
+    overflow: 'auto',
+  },
+  cardB: {
+    margin: '0rem',
+    padding: '0rem',
+  },
+  cardContent: {
+    marginLeft: '5rem',
+    marginRight: '5rem',
+    minWidth: 200,
+  },
+}));
```

Comment:
I think that, semantically, the html and css used in the old application where this component was originally designed, could be slightly improved for readability and accessibility. I'll try to offer some limited insights in the next comments. 

For this comment, just keep in mind that possibly some of the classes and names might need to be renamed for clarity, and quite possibly reworked for a different, more accessible markup.